crates/rpc/eth/transaction.rs

```diff
@@ -0,0 +1,69 @@
+use super::block::BlockIdentifier;
+use crate::utils::RpcErr;
+use ethereum_rust_core::{types::GenericTransaction, Bytes};
+use ethereum_rust_evm::{evm_state, ExecutionResult, SpecId};
+use ethereum_rust_storage::Store;
+use serde_json::Value;
+use tracing::info;
+
+pub struct CallRequest {
+    transaction: GenericTransaction,
+    block: Option<BlockIdentifier>,
+}
+
+impl CallRequest {
+    pub fn parse(params: &Option<Vec<Value>>) -> Option<CallRequest> {
+        let params = params.as_ref()?;
+        if params.len() != 2 {
+            return None;
+        };
+        Some(CallRequest {
+            transaction: serde_json::from_value(params[0].clone()).ok()?,
+            block: serde_json::from_value(params[1].clone()).ok()?,
+        })
+    }
+}
+
+pub fn call(request: &CallRequest, storage: Store) -> Result<Value, RpcErr> {
+    let block = request.block.clone().unwrap_or_default();
+    info!("Requested access list creation for tx on block: {}", block);
```

Comment:
Please update this message

Reply:
oops, I saw it in logs and was sure I corrected it (more than once 🤔  )